src/codeu/chat/DerbyStore.java

```diff
@@ -95,6 +102,43 @@ public DerbyStore() {
 		}
 	}
 	
+	@Override
+	public User getUser(Uuid userid) throws SQLException, IOException {
+		PreparedStatement getUser = conn.prepareStatement(getUserInfo);
```

Comment:
I don't quite know how PreparedStatement works: would it be more efficient to create these once at the beginning instead of every time the function is called? Or maybe cache them as you create them?